src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -435,7 +436,15 @@ static void uct_rdmacm_cm_handle_error_event(struct rdma_cm_event *event)
         }
 
         log_level = UCS_LOG_LEVEL_DEBUG;
```

Comment:
according to the PR tittle it should be as below, i.e. `uct_rdmacm_cm_ep_get_cm(cep)->super.config.failure_level;`

Reply:
it is aligned with master.
update master branch too?

Reply:
@evgeny-leksikov i think we don't consider "rejected" as real error since it can be part of normal flow, that's why level is debug

Reply:
@evgeny-leksikov ok to squash?

Reply:
yes